src/stores/config.ts

```diff
@@ -31,6 +31,9 @@ export const useConfigState = defineStore("config", {
         user: "",
         pass: "",
       },
+      others: {
+        marketplaceId: "",
```

Comment:
We can use the `marketplaceId` directly instead of nesting it inside the `others` object.

Reply:
Updated: [3f124e0](https://github.com/mehedimi/dokan-local/pull/2/commits/3f124e0877a507b60e3186feaafca3e3da1dc089)